backend/src/test/java/gov/cdc/usds/simplereport/idp/repository/DemoOktaRepositoryTest.java

```diff
@@ -408,17 +408,7 @@ void deleteOrgAndFacilities() {
         Set.of(ABC_2),
         Set.of(OrganizationRole.ENTRY_ONLY, OrganizationRole.ALL_FACILITIES),
         true);
-    _repo.deleteFacility(ABC_1);
 
-    OrganizationRoleClaims amos_expected =
-        new OrganizationRoleClaims(
-            ABC.getExternalId(),
-            Set.of(),
-            Set.of(OrganizationRole.NO_ACCESS, OrganizationRole.USER));
-
-    assertTrue(
-        new OrganizationRoleClaimsMatcher(amos_expected)
-            .matches(_repo.getOrganizationRoleClaimsForUser(AMOS.getUsername()).get()));
```

Comment:
part of cleanup of unused deleteFacility method

Reply:
as part of the `deleteOrganization` method we do remove the facilities from the `orgFacilitiesMap`. Maybe we can check that the `orgFacilitiesMap` for that org is `null` after deleting the organization on line `415`?

Reply:
I will add this in my next PR!